arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1936 @@
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/eventHubEndpoints/{eventHubEndpointName}/ConsumerGroups/{groupName}": {
+      "get": {
+        "tags": ["GET"],
+        "summary": "Get EventHub Consumer Group for an IotHub",
+        "description": "Get EventHub Consumer Group for an IotHub",
+        "operationId": "IotHubResource_GetEventHubConsumerGroup",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "Name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "eventHubEndpointName",
+          "in": "path",
+          "description": "Name of the event hub endpoint.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "groupName",
+          "in": "path",
+          "description": "Name of the consumer group.",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EventHubConsumerGroupInfo"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": ["PUT"],
+        "summary": "Add an EventHub Consumer Group to an IotHub",
+        "description": "Add an EventHub Consumer Group to an IotHub",
+        "operationId": "IotHubResource_CreateEventHubConsumerGroup",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "Name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "eventHubEndpointName",
+          "in": "path",
+          "description": "Name of the event hub endpoint.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "groupName",
+          "in": "path",
+          "description": "Name of the consumer group.",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
```

Comment:
Missing description. (Descriptions are important, they are used to generate the help documentation for the APIs)